### PR TITLE
Backend changes

### PR DESCRIPTION
**Pull request by DCalliet**
Noticed some inconsistencies in the way that data was being handled by the edit user function. Now sends JSON data easily readable by javascript. 

Also fixed a view sending bad data to the account page upon loading. 

EVERYTHING IS BETTER! MUCH BETTER!

### CODE REVIEW

**Comment by DCalliet** [comment]
@dborstelmann : look at it, I'm scared its all broken.
@piyushsprasad : look at it and admire its greatness. Nah, looking at some of the changes should give you a better grasp on whats going on in the app.

File: civiwiki/api/views_templates.py [hunk 1]
```diff
@@ -47,6 +47,7 @@ def dbview(request):
 def about_view(request):
 	return TemplateResponse(request, 'about.html', {})
 
+@login_required
 def account_home(request):
-	user_data = Account.objects.filter(user_id=request.user.id)[0].__dict__
-	return TemplateResponse(request, 'account_home.html', {'user_data': json.dumps({k: str(v) for k, v in user_data.items()})})
+	acc = Account.objects.get(user_id=request.user.id)
```
Comment (dborstelmann) [inline civiwiki/api/views_templates.py:52]: Did you test this?

Comment (DCalliet) [inline civiwiki/api/views_templates.py:52]: yeah, this works a billion times better than whoever wrote what I deleted.

Comment (dborstelmann) [inline civiwiki/api/views_templates.py:52]: 😒😒😒😒😒

**Pull request merged**